renderdoc/driver/gl/gl_resources.cpp

```diff
@@ -543,7 +547,7 @@ GLenum GetSizedFormat(const GLHookSet &gl, GLenum target, GLenum internalFormat,
   }
 
   GLint red, depth, stencil;
-  if(HasExt[ARB_internalformat_query2] && gl.glGetInternalformativ)
+  if(gl.glGetInternalformativ)
```

Comment:
I think I figured out what was happening, the glGetInernalformativ call was actually calling the emulated version and therefore works even without ARB_internalformat_query2.
Having said that, since I can no longer reproduce the assert I was getting from taking the sensible default code path, I'll just revert this change.